make-release.sh

```diff
@@ -184,63 +186,79 @@ replaceFieldSed()
   sed -i "${YAMLFILE}" -r -e "s#( ${YAMLKEY}: ).+#\1${YAMLVALUE}#"
 }
 
-versionUpdate() {
+minorVersionUpdate() {
```

Comment:
I am starting to think we should stop touching things in the main branch during releases.

We could:

* Keep only the HEAD branch in `antora-playbook-for-development.yml`

* Have `antora.yml` with static versions:

```
    prod-ver-major: "7"
    prod-ver-patch: "main"
    prod-ver: "main"
```

* Challenge the existence of the prod-prev-ver attribute and remove it.

Reply:
minorVersionUpdate() now unused, we can remove the function entirely.